### DISCUSSION TITLE
Gate Dimension order (x/y axis order)

### DISCUSSION THREAD

**Discussion by brianbourke75** [whitews/FlowKit#189] [Q&A] [answered]
When getting the dimensions that a 2D gate (Rectangle, Polygon, etc) is using, how does one tell which one is on the x-axis and which one is on the y-axis? Thanks!

**Comment by whitews** [comment, marked as answer]
Hi Brian,

The order of the dimensions corresponds to the order of the coordinates in each vertex in the PolygonGate. The RectangleGate ranges are defined in the Dimension instances themselves and not as vertices (the way GatingML specifies), but the order of the dimensions is basically [dim_x, dim_y].

You can see how this works in the notebook replicating a FlowJo gating tree here:

https://flowkit.readthedocs.io/en/latest/notebooks/advanced/replicate-flowjo-wsp.html#Define-Singlet-gate

Hope this helps!
-Scott
